tools/data-handler/src/validate.ts

```diff
@@ -514,6 +548,44 @@ export class Validate {
     return validationErrors.join('\n');
   }
 
+  /**
+   * Validate that long and short resource names are valid.
+   * @param resourceType Type of resource
+   * @param resourceName Name of resource
+   * @param prefixes currently used project prefixes
+   * @returns resource name as valid resource name (long format); throws in error cases.
+   */
+  public async validResourceName(
```

Comment:
Should we call the long name "name" and short name "identifier" or something similar? I think it would be more clear and it would be clear what function we wanna use

Reply:
Good point. I'll open a discussion on Teams about this. Let's see what the hive mind says.